### ISSUE TITLE
DropDeck - Web Frontend Lead - Remote/Saigon - FT

### ISSUE THREAD

**Issue by vovietanh** [awesome-jobs/vietnam#392]
## A Word About Us 
DropDeck provides a platform that helps investors find and assess potential startup investments much faster and accurately with our innovative Funnel Design and Trust System technology. The platform also helps startups prepare and arrange their information in the best way that investors expect. This is our best attempt to direct more investable capital ("spare money of rich people") into startups to fuel world innovations and improve our future quality of life.

## Location
* District 1, Saigon, Vietnam

## Salary Expectation
* ~ $1000/month
* Stock option (appraisal after 6 months, vesting for 1 year)

## You'd better have
* Knowledge of ReactJS
* Good understanding of Javascript, HTML5, CSS3, Responsive Design, Git, jQuery, Ajax
* Logical, analytic and algorithmic thinking
* Quick learning to pick up new technologies
* Obsessive attention to detail

### Why it would be awesome to work with us
* Chance to work with our terrific CTO (whom you'll meet for technical interview), train your algorithmic thinking, train to become a full-stack developer, employ technologies of the future, such as python, pycharm, django, etc.
* Chance to build an incredible product used by respected people - visionary investors and exponential founders aross Vitenam, Singapore, Hong Kong, Thailand, Japan, Finland and the US.
* Chance to be listened to and contribute to building a company where you enjoy working
* Chance to be our long term shareholder
* Accommodation at a founder's (my) house if you're not living in Saigon, Vietnam (I'm straight male)

### You will be responsible for
* Building our products (at godropdeck.com) to help users (startups, investors, agencies, partners) solve their problems more and more efficiently 
* Experimenting with your ideas to achieve the objective above
* Constantly testing, quality control and optimization to achieve the objective above

### What are hard problems that your company is trying to solve?
* The secret weapon we're building for our platform is a system to calculate people's level of trust - founders need trustworthy investors to bet on their future and vice versa.
* In this ever increasingly complicated world, "calculating trust" will be the most important and profitable thing. When we have perfected the "trust calculation" engine, it can be applied to ANY kind of business in the world (e-commerce, banking, logistics, AI, education, etc.).
* I have theorized that system a long time ago and my CTO has actually built a similar system to calculate how trustful a person is, in order to decide how much one can loan money and for how long.

### Contact
* Send your heartfelt message (or inquiries) and Elon@GoDropDeck.com (+84-903-72-62-88)

"May Elon be with you"

----

P.S. I wish to share a little more about myself and my motivation.

Circa 2008, as an environmental scientist, I was studying in depth how climate change (coupled with other social trends) will decrease our future quality of life and that there's not enough money being invested into innovations to resolve that future problem. I was invited to most environmental conferences around the world to present solutions. I was raising tens of thousands of euros in funding from the EU to fight climate change.

Out of $100 trillion of global investible capital (2x the amount of global GDP), only 1.15% was invested in innovations. Simply put, our future is in grave danger, and only a handful of top minds such as Elon Musk and Stephen Hawking realized that. 

As a result, my vision is a world where capital always flows into the right directions-innovations (AI, energy, cognitive augmentation, etc.) to maintain a sustainable future.

Ever since 2008, I was developing economic solutions that dealt with the global misallocation of capital, which I later realized was caused by the global misallocation of attention (how can you invest in something if you haven't heard about it or don't know how serious it is). In response to that realization, I was trying to build a knowledge platform to direct global attention in the right directions.  

I've been searching relentlessly for solutions to "save the future". I studied and presented about all exponential technologies for XPRIZE. I read about politics to understand how the world works. I wrote a sc-fi novel about physics, neuroscience, evolutionary biology, military and religion to spread futuristic ideas. I wrote a short story about how the world looks like in 2055. I wrote about the impact & meaning of Brexit and Trump. My journey so far is summarized here to show why what I'm doing NOW is important.

After engaging in the startup world, I quickly realized that I can achieve my initial vision by helping investors invest more into the best technologies/startups - directing all investible capital into future-improving innovations. And we have done much research to prove that we can, profitably.

**Comment by vovietanh** [comment]
We have successfully registered for AI XPRIZE and this is what we heard from Amir Banifatemi, AI XPRIZE Lead: "I checked their application and website. They have a good story and a good chance. Convey that to them." 

We have checked your profile on AI Portal and found that you're looking for a team to join. So I would like share with you about our project.

Our team is building a web platform for investors to source & evaluate venture deals in the fastest & most efficient way. When it comes to venture assessment, it comes down to determining how trustworthy each person involved is. Our core technology deals with calculating a trustworthiness (or "Trust") score for the investors and venture builders, based on metadata about their behavior on our website, their social accounts, other ranking/scoring systems such as Klout score, and potentially thousands of other factors such as credit history (from credit data providers such as CIC, Transunion).

In our definition, one’s (industry-specific) trust score (%) is the probability that one’s claim (on a specific matter) is true (if not binary then accurate above a certain threshold, say an allowed error margin of 10%).

In this ever increasingly complex world, "calculating trustworthiness" will be the most important and thus lucrative thing. When we have refined a "trust calculation engine", it can be applied to any kind of business (e-commerce, banking, logistics, HR, education, etc.), and it can take in visual, auditory, linguistic or any kind of inputs.

---

Our challenge is, though we identified AI  as the key from the beginning to optimize search, evaluation & matching of venture deals, we are in the earliest phase. We just started building our website and the infrastructure since December and moved into an office together early this month. 

We decided to enter this competition to accelerate our learning, product and personal development. Though we ourselves are working and studying as hard as we can, we are seeking for like-minded partners to fight in this world class arena. We are looking for diverse expertise, especially in data gathering, cleansing, modeling, and venture investment.

Right now we have only two members, me (business analysis, product management, basic ML knowledge), and Michael who is cc-ed (full stack development, data engineer). Michael has experience implementing a similar system to calculate how trustworthy a person is, in order to decide how much one can loan money and for how long (5000 features including metadata such as how long it takes an applicant to fill out the form, how many times you delete, whether you fill out all in one seating or more, etc.).

If you are interested, let's discuss more.
